go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/observatorium/api
 
-go 1.17
```

Comment:
This should not be needed as this declares the minimum version required by the project, however we don't use any go 1.18-specific features.